helsingborg-stad/app-mitt-helsingborg
# fix(updated-login-screen): update according to the new illustration

## Explain the changes you’ve made

Update login screen according to new design, with new background image and a red color scheme.

## Explain why these changes are made

To make it pretty!

## Explain your solution

Added the background illustration to assets (possibly we should have a new folder for onboarding and login things?), and then just use the ImageBackground component from ReactNative to set a background for the login screen. Also change colors of the buttons and some text to match the red color theme that we want. 

## How to test the changes?

1. Checkout this branch
2. Go to the login screen. Check that it looks nice; see if anything else needs changing.

## Was this feature tested in the following environments?
- [] Storybook on a iOS device/simulator.
- [] Building the Application on a iOS device/simulator.

## Screenshots
![new login](https://user-images.githubusercontent.com/7421890/105329769-cf59e680-5bd1-11eb-8e69-cef5807bd91d.png)
